examples/sctk-drag/src/main.rs

```diff
@@ -0,0 +1,63 @@
+use iced::{
+    wayland::InitialSurface, widget, window, Application, Command, Element,
+    Subscription, Theme,
+};
+
+fn main() {}
```

Comment:
This example isn't doing anything currently.

Reply:
Oh, I ended up just doing the example in todos_sctk :sweat_smile: I think I was planning on making this a custom DnD example outside of a widget implementation. Should I fill out this example then?

Reply:
ahh and for custom DnD maybe the dnd_listener could still be useful and filled out more.